store/tikv/lock_resolver.go

```diff
@@ -368,40 +358,77 @@ func (lr *LockResolver) GetTxnStatus(txnID uint64, callerStartTS uint64, primary
 	if err != nil {
 		return status, err
 	}
-	return lr.getTxnStatus(bo, txnID, primary, callerStartTS, currentTS)
+	return lr.getTxnStatus(bo, txnID, primary, callerStartTS, currentTS, true)
```

Comment:
We should not use `true` here.

Reply:
Is it only used in test?

Reply:
It is used by other tools.

Reply:
https://github.com/pingcap/tidb-binlog/blob/cd006abe2f5af164e30f19d2a50b031d1e0f2bfc/pump/storage/storage.go#L490

Reply:
After discussion we will keep it true.